reports/filesDataExporter/batch/batch_test.go

```diff
@@ -0,0 +1,309 @@
+package batch
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	strfmt "github.com/go-openapi/strfmt"
+	"github.com/golang/mock/gomock"
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog"
+
+	"github.com/iryonetwork/wwm/gen/storage/models"
+	"github.com/iryonetwork/wwm/reports/filesDataExporter"
+	"github.com/iryonetwork/wwm/reports/filesDataExporter/mock"
```

Comment:
could not import github.com/iryonetwork/wwm/reports/filesDataExporter/mock (cannot find package "github.com/iryonetwork/wwm/reports/filesDataExporter/mock" in any of:
	/tmp/tmp.OXDHvmiU1p/src/github.com/iryonetwork/wwm/vendor/github.com/iryonetwork/wwm/reports/filesDataExporter/mock (vendor tree)
	/usr/local/go/src/github.com/iryonetwork/wwm/reports/filesDataExporter/mock (from $GOROOT)
	/tmp/tmp.OXDHvmiU1p/src/github.com/iryonetwork/wwm/reports/filesDataExporter/mock (from $GOPATH))